### PR TITLE
feature(ui-ux): add USD values for min. next bid

### LINKED ISSUE #1985: Auction Card - Min. next bid
On Auction cards, we need to put the prices based on DEX values on the Min. next bid. Check epic for more info #1856 
image

### LINKED ISSUE #1986: Auction Detail Screen - Min. next bid
On Auction - Auction detail page, we need to add USD values (based on DEX) on the Min. next bid, both on the auction tab and the card below. More details, please check epic #1856 
image

### LINKED ISSUE #1987: Quick bid card, Min next bid and balance
Add USD values on Quick Bid card of Auctions details and Auction Card - Quick Bid - Bottom Sheet
image

### PR DESCRIPTION
**Pull request by kyleleow**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- To display USD values based on DEX prices for min. next bid in auction-related screens
- To display USD values for current balance in quick bid bottom sheet

#### Which issue(s) does this PR fixes?:

Fixes #1985
Fixes #1986 
Fixes #1987 

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [x] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

**Comment by fullstackninja864** [comment]
@kyleleow There is inconsistency in the price for auctions Please check SS
image

**Comment by thedoublejay** [comment]
@fullstackninja864 I think @kyleleow calculation is correct. You can manually calculate it

**Comment by kyleleow** [comment]
> @kyleleow There is inconsistency in the price for auctions Please check SS image

DeFiScan is using oracle price while we calculate using DEX prices, will inform the team
https://github.com/DeFiCh/scan/blob/6d1ae70e11442a16ae31fc36ec7cdb51acd713d2/src/pages/auctions/_components/commons/BidAmountValue.tsx#L23

File: mobile-app/app/screens/AppNavigator/screens/Auctions/components/MinNextBidTextRow.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            >
+              {value}
+            </ThemedText>
+          )}
+          thousandSeparator
+          value={props.minNextBidInToken}
+        />
+        <ActiveUSDValue
+          price={new BigNumber(props.minNextBidInUSD)}
+          testId={props.testID !== undefined ? `${props.testID}_usd` : props.testID}
```
Comment (fullstackninja864) [inline mobile-app/app/screens/AppNavigator/screens/Auctions/components/MinNextBidTextRow.tsx:59]:
```suggestion
          {...(props.testID !== undefined && { testId: `${props.testID}_usd` })}

```

File: mobile-app/app/screens/AppNavigator/screens/Auctions/components/QuickBid.tsx [hunk 2]
```diff
@@ -57,7 +63,7 @@ export const QuickBid = ({
         token: Number(loanTokenId)
       },
       displaySymbol: loanTokenDisplaySymbol,
-      onBroadcast: () => {}
+      onBroadcast: () => { }
```
Comment (fullstackninja864) [inline mobile-app/app/screens/AppNavigator/screens/Auctions/components/QuickBid.tsx:66]:
```suggestion
      onBroadcast: () => {}
```

**Review by pierregee — DISMISSED** [review]

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**